config/routes.rb

```diff
@@ -1,3 +1,6 @@
 # frozen_string_literal: true
 Rails.application.routes.draw do
+  resources :buildings
+  resources :suites
+  resources :rooms
```

Comment:
I'm not 100% confident that it would be a common use case, but I'm also not sold on having to deal with a nested resource. What would the other benefits of nesting be?